src/hooks/constants/graphql/FetchConstants.query.graphql

```diff
@@ -0,0 +1,3 @@
+query FetchConstants {
```

Comment:
what's the point in providing an empty query definition, if it isn't consumed anywhere in the PR?

Reply:
I saw it done elsewhere and imitated,  I think I thought it needed to be there for `codegen` or something. 

example: https://github.com/galacticcouncil/Basilisk-ui/blob/develop/src/hooks/accounts/graphql/GetAccountById.query.graphql

that's not used anywhere ☝️ 

So solution here is to 
- delete `FetchConstants.query.graphql` file.  
- Would you like me to delete other files like this that are not used in codebase as part of this PR ?

Reply:
I think we should leave out the query, but not the type definition for it - since that dictates how the resolver should behave. We will write a query when it comes to it - in the containers.